tests/integration_tests/functional/test_api.py

```diff
@@ -1013,6 +1018,11 @@ def test_api_memory_hotplug(uvm_plain):
     status = test_microvm.api.memory_hotplug.get().json()
     assert status["total_size_mib"] == 1024
 
+    # Patch API should work after boot
+    test_microvm.api.memory_hotplug.patch(requested_size_mib=512)
+    status = test_microvm.api.memory_hotplug.get().json()
+    assert status["requested_size_mib"] == 512
+
```

Comment:
should we add a test that we can't `PATCH` we a `requested_size_mib` that reduces the available memory? (and check that it fails)